sql/README.md

```diff
@@ -20,6 +20,8 @@ Once you are ready to add a new query to the repository, open a pull request fol
 
 ### 2023/03
 * [Top class names used on lazy loaded LCP images](./2023/03/top-lazy-lcp-class-names.sql)
+* [% of WordPress sites that do not implement Critical CSS via custom metrics](./2023/03/critical-css-opportunity-custom-metics.sql)
+* [% of WordPress sites that do not implement Critical CSS via custom metrics (using `$._renderBlockingCSS`)](./2023/03/critical-css-opportunity-custom-metics-alternative.sql)
```

Comment:
There is a typo in the file names (`metics` --> `metrics`). Please rename the files accordingly and update the links here.

Reply:
Done.